web/sites/default/_docker.settings.local.php

```diff
@@ -0,0 +1,53 @@
+<?php
```

Comment:
Why this file... it also located under web/sites/default/docker.settings.local.php which is the right one?